morpheus/_lib/messages/module.cpp

```diff
@@ -379,13 +378,50 @@ PYBIND11_MODULE(messages, _module)
              py::arg("config"))
         .def("config", pybind11::overload_cast<ControlMessage&>(&ControlMessageProxy::config))
         .def("copy", &ControlMessageProxy::copy)
-        .def("get_metadata", &ControlMessageProxy::get_metadata, py::arg("key") = py::none())
+        .def(
+            "get_metadata",
+            [](ControlMessage& self, const py::object& key = py::none(), const py::object& default_value = py::none()) {
```

Comment:
Dont specify the default values for the lambda function. Only use the `py::arg() = default` syntax